src/main/java/frc/robot/subsystems/vision/Camera.java

```diff
@@ -24,40 +24,53 @@
 import edu.wpi.first.math.util.Units;
 
 public class Camera {
-
-  public final String name;
-  public final String path;
-  public final Transform3d transform;
-  public final CameraIntrinsics intrinsics;
-
-  public final PhotonCamera camera;
-  public final PhotonPoseEstimator estimator;
-
-  private final double MAX_PITCHROLL = Units.degreesToRadians(5);
-  private final double MAX_Z = Units.inchesToMeters(7);
-
-  private final double xyStdDevCoefficient = 0.08;
-  private final double thetaStdDevCoefficient = 0.16;
-  private final double coefficientFactor = 10_000.0;
+  private String path;
+  private Transform3d transform;
+  private CameraIntrinsics intrinsics;
+  private AprilTagFieldLayout field;
+  private PhotonCamera camera;
+  
+  private final PhotonPoseEstimator estimator;
+  
+  private final double MAX_PITCHROLL_DEGREES = 5;
+  private final double MAX_PITCHROLL = Units.degreesToRadians(MAX_PITCHROLL_DEGREES);
+  private final double MAX_Z_INCHES = 7;
+  private final double MAX_Z = Units.inchesToMeters(MAX_Z_INCHES);
+
+  private final double XY_STD_DEV_COEFFICIENT = 0.08;
+  private final double THETA_STD_DEV_COEFFICIENT = 0.16;
+  private final double COEFFICIENT_FACTOR = 10_000.0;
+
+  private final String PATH_VISION = "Vision/";
+  private final String PATH_TAG_POSES = "/TagPoses";
+  private final String PATH_CAM_TRANSFORM = "/CamTransform";
+  private final String PATH_EST_POSE_UNFILTERED = "/EstPoseUnfiltered";
+  private final String PATH_XY_STD_DEV = "/XyStdDev";
+  private final String PATH_THETA_STD_DEV = "/ThetaStdDev";
+  private final String PATH_NUM_TARGETS = "/NumTargets";
+  private final String PATH_AVG_TAG_AREA = "/AvgTagArea";
+  private final String PATH_EST_POSE = "/EstPose";
```

Comment:
part of me's a bit iffy about constantizing path names, but they're used more than once, so I think I'm fine with it specifically in this case.